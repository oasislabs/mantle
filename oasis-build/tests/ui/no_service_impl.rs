use oasis_std::Context;

#[derive(oasis_std::Service)]
pub struct Counter(u32);

fn main() {
    oasis_std::service!(Counter);
}
