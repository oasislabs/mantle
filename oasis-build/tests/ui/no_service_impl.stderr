error: Missing definition of `Counter::new`.
 --> $DIR/no_service_impl.rs:4:1
  |
4 | pub struct Counter(u32);
  | ^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

